package.json

```diff
@@ -179,5 +180,14 @@
       "browser",
       "mocha"
     ]
+  },
+  "stylelint": {
+    "rules": {
+      "block-no-empty": null,
+      "color-no-invalid-hex": true,
+      "declaration-colon-space-after": "always",
+      "indentation": "tab",
+      "max-empty-lines": 2
+    }
```

Comment:
These rules look good to me, just asking to confirm if there's been a discussion with the maintainers regarding them.

Reply:
I based the indentation rules on zulip webapp. I also intend to use the rules from the webapp once, these basic rules get merged.